trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/wing_planform.py

```diff
@@ -111,8 +111,6 @@ def wing_planform(wing):
         affected_area           = (wing_chord_flap_start + wing_chord_flap_end) * (flap.span_fraction_end- flap.span_fraction_start)*span / 2.          
          
     # update
-    wing.chords.root                = chord_root
```

Comment:
Why were these removed?

Reply:
Because to me, tip and root chord are more easily known than taper and aspect ratio, definitely in the case of multi-segmented wings where the definition of taper and aspect ration can lead to ambiguity in the multiple segments. Therefore in my opinion, when running this script, even though the user probably defined tip and root chord, they are updated without the user's knowledge which can lead to the incorrect result. So we could include a flag if the user wants to update the tip and root chord or not.

Reply:
Okay, we'll need to discuss this with the larger group.